modules/serverpod_auth/serverpod_auth_server/test/firebase/firebase_auth_mock.dart

```diff
@@ -0,0 +1,243 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:jose/jose.dart';
+import 'package:mockito/mockito.dart';
+import 'package:openid_client/openid_client.dart';
+import 'package:openid_client/openid_client_io.dart';
+import 'package:serverpod_auth_server/src/firebase/auth/auth_http_client.dart';
+import 'package:rsa_pkcs/rsa_pkcs.dart';
+
+class MockAuthBaseClient extends Mock implements AuthBaseClient {
```

Comment:
Question: What functionality from `Mock` are we actually relying on?

Couldn't we just remove the Mock dependency?